scripts/populate_ingest/convert/conversion_utils.py

```diff
@@ -0,0 +1,55 @@
+import re
+
+
+ACCESSION_PATTERNS = {
+    "insdc_project_accessions": "^[D|E|S]RP[0-9]+$",
+    "ega_accessions": "EGA[DS][0-9]{11}",
+    "dbgap_accessions": "phs[0-9]{6}(\\.v[0-9])?(\\.p[0-9])?",
+    "geo_series_accessions": "^GSE.*$",
+    "array_express_accessions": "^E-....-.*$",
+    "insdc_study_accessions": "^PRJ[E|N|D][a-zA-Z][0-9]+$",
+    "biostudies_accessions": "^S-[A-Z]{4}[0-9]+$"
+}
+
+def get_accessions(data_accessions: str):
+    accessions = {}
+    for accession in data_accessions.split(','):
+        accession = accession.strip()
+        for key, pattern in ACCESSION_PATTERNS.items():
+            regex = re.compile(pattern)
+            if regex.match(accession):
+                accessions.setdefault(key, []).append(accession)
+    return accessions
+
+# ToDo: Move this to json-converter
```

Comment:
Please don't leave todos in the code. Either do or delete.

Reply:
This to-do was already there as part of the code, before I added things to it.

I'm unaware of the context of this comment, so I can't do or delete it.

Reply:
If you don't know what it is, and cannot contact the person who wrote this todo, delete it.
It should not have been there in the first place.